### ISSUE TITLE
Return value of Sentry\Laravel\ServiceProvider::getUserConfig() must be of the type array

### ISSUE THREAD

**Issue by gilbitron** [getsentry/sentry-laravel#301]
Hi. I'm getting the following error while trying to install Sentry in a Laravel 6.x app on PHP 7.3 on a Forge server:

```
> @php artisan package:discover --ansi

   Symfony\Component\Debug\Exception\FatalThrowableError  : Return value of Sentry\Laravel\ServiceProvider::getUserConfig() must be of the type array, null returned

  at .../vendor/sentry/sentry-laravel/src/Sentry/Laravel/ServiceProvider.php:204
    200|      * @return array
    201|      */
    202|     private function getUserConfig(): array
    203|     {
  > 204|         return $this->app['config'][static::$abstract];
    205|     }
    206| 
    207|     /**
    208|      * Get the services provided by the provider.

  Exception trace:

  1   Sentry\Laravel\ServiceProvider::getUserConfig()
      .../vendor/sentry/sentry-laravel/src/Sentry/Laravel/ServiceProvider.php:56

  2   Sentry\Laravel\ServiceProvider::register()
```

**Comment by stayallive** [comment]
Try running `php artisan cache:clear`. That should fix it.

**Comment by gilbitron** [comment]
Clearing the route + config cache before running composer install fixed it 👍

**Comment by stayallive** [comment]
Clearing the config cache should be enough, but I also added a safeguard to prevent this from happening which will be released with 1.5 to make sure this errors doesn't brick your app 😄

**Comment by gilbitron** [comment]
Awesome 👍
